php/src/Google/Protobuf/Internal/FileDescriptorProto.php

```diff
@@ -400,6 +418,7 @@ public function hasExtension()
 
     /**
      * <code>optional .google.protobuf.FileOptions options = 8;</code>
+     * @return Google\Protobuf\Internal\FileOptions
```

Comment:
The tags seem inconsistent about whether or not to include a leading `\` for fully qualified types. I think that we do want to include a leading `\` in all cases